### DISCUSSION TITLE
Increase Partitions do not work

### DISCUSSION THREAD

**Discussion by mordp1** [joschi/kafka-gitops#175] [Q&A]
Hello!
I test with the [Release 0.2.16](https://github.com/joschi/kafka-gitops/releases/tag/0.2.16) and [Release 0.3.0](https://github.com/joschi/kafka-gitops/releases/tag/0.3.0).

From `state.yaml` config : 

```
topics:
  my-example-topic:
    partitions: 4
    replication: 3
    configs:
      retention.ms: 3600000
```
After increasing the number of partitions from 4 to 6 for example

```
$ kafka-gitops -v --no-delete --skip-acls -f state.yml apply
Executing apply...

19:45:17.847 [main] INFO com.devshawn.kafka.gitops.config.KafkaGitopsConfigLoader - Kafka Config: {bootstrap.servers=kafka:9092}
19:45:17.852 [main] INFO com.devshawn.kafka.gitops.service.ConfluentCloudService - Using ccloud executable at: ccloud
19:45:17.853 [main] INFO com.devshawn.kafka.gitops.service.ParserService - Parsing desired state file...
19:45:22.976 [main] INFO com.devshawn.kafka.gitops.manager.PlanManager - [PLAN] Topic my-example-topic exists, it will not be created.
19:45:22.981 [main] INFO com.devshawn.kafka.gitops.manager.PlanManager - [PLAN] Topic my-example-topic | [NO_CHANGE] retention.ms
[SUCCESS] There are no necessary changes; the actual state matches the desired state.
```
As far as I can see, nothing happened, no matter what number I put in the partitions and replication, the plan and apply return: `[SUCCESS] There are no necessary changes; the actual state matches the desired state.`
As far as I can see this is not true.

I use Apache Kafka version 3.4 and not Confluent.

Should I use a different configuration?
I would like just to create and maintain topics in the Kafka Cluster.
The creation of the topics is OK.

**Comment by joschi** [comment]
Changing the number of partitions is currently not supported by kafka-gitops.

While @jrevillard contributed this functionality to the original project in https://github.com/devshawn/kafka-gitops/pull/52, I had to revert the changes because they broke the tests in this project.

Refs https://github.com/devshawn/kafka-gitops/issues/34

**Comment by jrevillard** [comment]
Hello,

Just to explain that it broke the tests certainly because I also modified the plan output format in the same huge MR. Unfortunately I have no time to split both.The schema registry functionality is also using the new format (another Huge MR).

For info we use both functionalities in production since years now.

Best,
Jerome
